tests/Avalonia.Base.UnitTests/Utilities/InlineDictionaryTests.cs

```diff
@@ -8,6 +8,17 @@ namespace Avalonia.Base.UnitTests.Utilities;
 
 public class InlineDictionaryTests
 {
+    [Fact]
+    public void Set_Twice_With_Internal_Array_Works()
```

Comment:
Nit: we don't have an internal array yet, the name should be something like `Set_Twice_With_Single_Item_Works()`.